blocklylib-core/src/main/java/com/google/blockly/utils/ColorUtils.java

```diff
@@ -34,9 +34,9 @@
     public static final int ALPHA_OPAQUE = 255;
 
     public static final int DEFAULT_BLOCK_HUE = 0;
```

Comment:
Nothing is using this in this project any more, but I haven't removed it as it's public so it's possible other projects are depending on it.